packages/ndk-multilib/build.sh

```diff
@@ -37,11 +37,27 @@ prepare_libs() {
 	fi
 
 	cp $LIBATOMIC/libatomic.a $TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/$SUFFIX/lib/
+
+	cp $LIBATOMIC/libunwind.a $TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/$SUFFIX/lib/
+}
+
+add_cross_compiler_rt() {
+	# FIXME: Should we get libcompiler-rt from libllvm package?
+	RT_PREFIX=$NDK/toolchains/llvm/prebuilt/linux-x86_64/lib64/clang/*/lib/linux
+	LIBLLVM_VERSION=$(. $TERMUX_SCRIPTDIR/packages/libllvm/build.sh; echo $TERMUX_PKG_VERSION)
+	RT_TMP=$TERMUX_PKG_TMPDIR/cross-compiler-rt
+	RT_PATH=$TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/lib/clang/$LIBLLVM_VERSION/lib/android/
+	mkdir -p $RT_TMP
+	cp $RT_PREFIX/* $RT_TMP/ || true
+	rm -rf $RT_TMP/*$TERMUX_ARCH*
```

Comment:
This removal seems to prevent the package from being platform-independent. Is this necessary? What happens if not removed?

Reply:
Some of them are provided by package `libcompiler-rt`. If we don't remove them, `dpkg -i` will fail to install this package due to file conflict.

Reply:
Then I propose that `libclang_rt.*`s be installed to another directory, and that `postinst` script be used to symlink to them (except for the ones targeting the native arch).

Reply:
And then this package can be platform-independent again. That sounds great!